src/app/components/progress-table/progress-table.component.html

```diff
@@ -170,14 +171,39 @@
         </ion-button>
       </ng-template>
     </ngx-datatable-column>
-  </ngx-datatable>
 
-  <ion-button color="primary" fill="clear" (click)="switchType()">
-    <ng-container *ngIf="type === 'team'">
-      VIEW PROGRESS BY STUDENT
-    </ng-container>
-    <ng-container *ngIf="type === 'student'">
-      VIEW PROGRESS BY PROJECT
-    </ng-container>
-  </ion-button>
+    <ngx-datatable-footer style="position: fixed; bottom: 0px;" *ngIf="true">
+      <ng-template
+        ngx-datatable-footer-template
+        let-rowCount="rowCount"
+        let-pageSize="pageSize"
+        let-selectedCount="selectedCount"
+        let-curPage="curPage"
+        let-offset="offset">
```

Comment:
why do we need those? `curPage, pageSize, selectedCount, offset`

Reply:
we need `pageSize` and `curPage` others removed. 
we need those two for pagination work. we have added pagination separately, to work pagination it need those two values.